src/Command/CacheClearCommand.php

```diff
@@ -54,7 +54,7 @@ public function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionPar
             ->addArgument('engine', [
                 'help' => 'The cache engine to clear.' .
                     'For example, `cake cache clear _cake_model_` will clear the model cache' .
```

Comment:
I'm not sure these are concatenating with a space. Leading to conjoined sentences.

Reply:
Needs periods and spaces.